src/products/firefox_desktop/fluent_migrations.md

```diff
@@ -50,23 +48,20 @@ The last step here is to make sure that the current sync process has completed:
 * Clone the repository locally and follow the instructions available in the [README](https://github.com/flodolo/fluent-migrations/blob/master/README.md) to set the configuration file.
-* Add the new migration recipes in the `recipes` folder. The script will look for any Python file starting with `bug_` in that folder, so it’s possible to run multiple migrations in one pass. More information about the folder’s structure are available in the repository’s README.
+* Add the new migration recipes in the `recipes` folder. The script will look for any Python file starting with `bug_` in that folder, so it’s possible to run multiple migrations in one pass. More information about the folder’s structure is available in the repository’s README.
 
 At this point, while each migration has been tested as part of the review before landing, it’s always good to run the migration against one locale, e.g. `it`:
-* Update the locale’s repository, check the results of `compare-locales`. Assuming l10n repositories are cloned in `~/migrations/locales`, and the quarantine repository in `~/migrations/gecko-strings-quarantine`, the command to run is `compare-locales ~/migrations/gecko-strings-quarantine/_configs/browser.toml ~/migrations/locales it`. Save the output in order to compare it with the results after the migration.
+* Update the locale’s repository, check the results of `compare-locales`. Assuming the l10n repository is cloned in `~/migrations/firefox-l10n`, and the quarantine repository is in `~/migrations/firefox-l10n-source` and the `update` branch is checked out, the command to run is `compare-locales ~/migrations/firefox-l10n-source/_configs/browser.toml ~/migrations/firefox-l10n it`. Save the output in order to compare it with the results after the migration.
 * Run the migration, without pushing, only for `it`: `./scripts/migrate it wet-run`. Then run compare-locales again, and check if the results are as expected: migrated strings should not appear as missing anymore, and there should be no errors.
-* If everything looks as expected, re-run the script adding the push option `./scripts/migrate it wet-run push`.
 
-### Push updates to gecko-strings
+### Push updates to the main branch of firefox-l10n-source
 
-Before [pushing updates](review.md#push-reviewed-strings-to-gecko-strings) to the official [gecko-strings repository](https://hg.mozilla.org/l10n/gecko-strings), make sure that:
-* The new strings and migrations are available in the [quarantine repository](https://hg.mozilla.org/l10n/gecko-strings-quarantine).
-* There are [no pending issues](#ensure-there-are-no-blocking-issues-in-gecko-strings-quarantine) that prevent the current content to be exposed in Pontoon.
+Before [merging updates](review.md#merge-reviewed-strings-to-main-branch-of-firefox-l10n-source) to the `main` branch of `firefox-l10n-source`, make sure that:
+* The new strings and migrations are available in the `update` branch.
```

Comment:
Migrations are not in firefox-l10n-source. What did you mean here?

Reply:
I've added some clarifying language to avoid confusion. This is the step to merge source strings (i.e. `update` into `main` on `firefox-l10n-source`).